main.sh

```diff
@@ -69,6 +69,8 @@ download_and_extract() {
     case "${tool}" in
         # xbuild's binary name is "x", as opposed to the usual crate name
         xbuild) installed_bin=("${bin_dir}/x") ;;
+        # editorconfig-checker's binary name is renamed below
+        editorconfig-checker) installed_bin=("${bin_dir}/${tool}") ;;
```

Comment:
It seems editorconfig-checker binary is usually named `ec`? https://github.com/editorconfig-checker/editorconfig-checker?tab=readme-ov-file#what
if so, this should be "${bin_dir}/ec{exe}".

Reply:
`go install github.com/editorconfig-checker/editorconfig-checker/v2/cmd/editorconfig-checker@latest` installs `editorconfig-checker`

homebrew, nix and arch use `editor-checker` https://repology.org/project/editorconfig-checker/versions

I'm not sure about alpine.  But in general distros are unlikely to let an obscure tool like this have a two letter bin name.

Reply:
Thanks for the info. Agreed that "editorconfig-checker" is a reasonable binary name here.

alpine: "package name" is "editorconfig-checker", but "binary name" is "ec": https://pkgs.alpinelinux.org/contents?branch=v3.14&name=editorconfig%2dchecker&arch=x86_64&repo=community

arch: "package name" is "editorconfig-checker", but "binary names" are both "ec" and "editor-checker": https://archlinux.org/packages/extra/x86_64/editorconfig-checker/ (click "Package Contents" section to show list)

homebrew: both "package name" and "binary names" are "editorconfig-checker": https://github.com/Homebrew/homebrew-core/blob/675ca659ebe69e0829d59104c2b74089e0acf6f6/Formula/e/editorconfig-checker.rb#L31

nix: 
- NixOS/nixpkgs: IIUC both "package name" and "binary names" are "editorconfig-checker": https://github.com/NixOS/nixpkgs/blob/release-23.11/pkgs/development/tools/misc/editorconfig-checker/default.nix
- editorconfig-checker/editorconfig-checker/default.nix: "package name" is "editorconfig-checker", but "binary names" are both "ec" and "editor-checker" (ec is symlink): https://github.com/editorconfig-checker/editorconfig-checker/blob/0e32afcec97ba925d3c5f6530186159e3f37bcb0/default.nix#L19

Reply:
Also is looks like they are going to discontinue using `ec` https://github.com/editorconfig-checker/editorconfig-checker/pull/255